### PR TITLE
add basic 1.127 support, RC4 property and some fix 1.124+

### PR DESCRIPTION
**Pull request by JVirant**
I fixed some issue that I created from previous commits especially with the encryption.

### CODE REVIEW

File: GameServer/serverproperty/ServerProperties.cs [hunk 1]
```diff
@@ -121,6 +121,12 @@ public static void InitProperties()
 		[ServerProperty("system", "client_version_max", "Maximum Client Version - Edit this to change which client version at the most have to be used: -1 = any, 1.80 = 180", -1)]
 		public static int CLIENT_VERSION_MAX;
 
+		/// <summary>
+		/// Enable RC4 encryption for communication between the client and the server
+		/// </summary>
+		[ServerProperty("system", "client_enable_encryption_rc4", "Enable client RC4 encryption - Advanced user only, you need a special launcher to enable encryption", false)]
```
Comment (Lotendan) [inline GameServer/serverproperty/ServerProperties.cs:127]: I was actually worried about the launcher, but you mention it properly here ;)

File: GameServer/packets/Server/PacketLib1124.cs [hunk 2]
```diff
@@ -3134,80 +3133,34 @@ public virtual void SendRealm(eRealm realm)
 				SendTCP(pak);
 			}
 		}
-		public virtual void SendRegions()
+		public virtual void SendRegions(ushort regionId)
```
Comment (Lotendan) [inline GameServer/packets/Server/PacketLib1124.cs:3136]: Seeing that we are only sending one region now, should we make this singular?
`SendRegion`?

Comment (JVirant) [reply GameServer/packets/Server/PacketLib1124.cs:3136]: I stay conservative about changes on public methods to keep a better compatibility with existent scripts or custom code.

**Comment by NetDwarf** [comment]
Works like a charm. Thank you! :smiley:

**Comment by JVirant** [comment]
> Works like a charm. Thank you! 😃

It doesn't... :(
I got a crash when I was with 2 clients : one 1.125d and another one 1.127d but it's a start!

**Comment by NetDwarf** [comment]
I should maybe say works like advertised :smile:

**Pull request merged**